models/services.ts

```diff
@@ -9,6 +9,8 @@ export interface Service {
   address?: string[] // Airtable "physical_addresses" table ids
   phoneNumbers?: string[]
   taxonomyString?: string[]
+  organizationNames?: string[]
+  organizationUrls?: string[]
```

Comment:
Added these fields to the `services` table in Airtable to avoid additional API calls to organizations' info since we can't join tables like in SQL